### DISCUSSION TITLE
rich presence delay in steamcommunity web

### DISCUSSION THREAD

**Discussion by johnnyHoog** [gamer2810/steam-miniprofile#11] [Show and tell]
I use miniprofile to get rich presence.but now there have a delay in the URL of user's Steam community profile page. There will be a delay of about 10 minutes. This makes this function seem very useless.The user's rich Presence can no longer be updated in real time.
In addition, I also found that only the rich status data of the steam profile page on the web side is delayed. The rich Presence data in the steam client friend panel are all real-time, but it seems very useless if you can only see the rich status of your friends. Of course, in order to obtain the personal rich status of any user we specify。Author, do you have any solution?Thank you for your help.

**Comment by gamer2810** [comment]
It seems like you are facing two problems:

1. 10 minutes delay when loading miniprofiles 
- Can you check if there is any caching on your website? I did not impose any cache on my service. It should update in real time with Steam's data.

2. Rich presence not showing
- When I write the README in 2021, only friends can see rich presence. I tested it today and it's no longer the case. As long as a user sets their `Game Details` privacy status to `PUBLIC`, `steam-miniprofile` can show it.
- I will change the README content :D
